### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
While working on antsibull-docs, I found some docs issues which will soon have an impact on docs rendering.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
collection docs

### CODE REVIEW

**Review by briantist — APPROVED** [review]

**Review by nilsding — APPROVED** [review]

**Comment by nejch** [comment]
LGTM

(at least for the gitlab modules :) )
image
--> 
image

**Review by samccann — APPROVED** [review]

**Review by mikemoerk — APPROVED** [review]

File: changelogs/fragments/licenses.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "All software licenses are now in the ``LICENSES/`` directory of the collection root (https://github.com/ansible-collections/community.crypto/pull/491)."
```
Comment (gotmax23) [inline changelogs/fragments/licenses.yml:2]: This looks like a mistake

Comment (felixfontein) [reply changelogs/fragments/licenses.yml:2]: It definitely is! Sometimes `git stash push/pop` is confusing me... I was working on a licensing PR, then got interrupted for some time, and forgot about it when starting on this one... unti noticing that there were waaaay to many changes. Let me fix that.

**Comment by felixfontein** [comment]
@briantist @nilsding @nejch @samccann @mikemoerk @gotmax23 thanks a lot to all of you for reviewing this! :)

**Pull request merged**